sky/engine/core/script/dart_init.cc

```diff
@@ -194,6 +194,28 @@ Dart_Handle GetVMServiceAssetsArchiveCallback() {
       ::dart::observatory::observatory_assets_archive_len);
 }
 
+static const char* kStdoutStreamId = "Stdout";
+static const char* kStderrStreamId = "Stderr";
```

Comment:
`*  ->  []`

This is just a nit.  It makes the static data slightly better in the binary.